wallet/bindings/java/lib/native/src/lib.rs

```diff
@@ -1,18 +1,28 @@
 // Copyright 2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
+#[cfg(target_os = "android")]
+#[macro_use]
+extern crate log;
+#[cfg(target_os = "android")]
+extern crate android_logger;
+
```

Comment:
You shouldn't need these lines at all

Reply:
I think was used to discover the problems with the `listen` function, but right now the Android `logcat` is too verbose, we could remove it now imo. This is the Brord's code, but I can remove it ofc.

Reply:
I'm just speaking of the `extern crate` which is not needed anymore in Rust. We can keep the ligger if you think it can be useful.